pageserver/pagebench/src/cmd/basebackup.rs

```diff
@@ -92,10 +92,8 @@ async fn main_impl(
     for timeline in &timelines {
         js.spawn({
             let timeline = *timeline;
-            // FIXME: this triggers initial logical size calculation
-            // https://github.com/neondatabase/neon/issues/6168
             let info = mgmt_api_client
-                .timeline_info(timeline.tenant_id, timeline.timeline_id)
+                .timeline_info(timeline.tenant_id, timeline.timeline_id, None)
                 .await
```

Comment:
frequent nit of mine: at this callsite, it's completely unclear what `None` means, unless you have inlay hints from your IDE that show the parameter name.

I'd always go with an custom enum, or flags, instead of the `Option<bool>`.
I.e., 

```rust
enum PrioritizeInitialSizeCalculationFlag {
    Dont,
    Do,
}
```

Reply:
Agreed. Made the change